src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceImplTest {
+
+    private static final User VALID_USER = new User("Ivanovich", "3455667",18);
+        private static final User TWO_VALID_USER = new User("Sidorovich", "34jkl667",33);
+        private static final User SAME_LOGIN_VALID_USER = new User("Ivanovich", "pp477777",85);
+
+        private static final User NULL_USER = new User(null, null,null);
```

Comment:
```suggestion
        private static final User nullUser = new User(null, null,null);
```
так як клас юзер не іммютабл потрібно використовувати стандартний неймнінг